mmc-google-analytics/src/main/java/info/novatec/metricscollector/google/aqeblog/AqeBlogReportRequestBuilder.java

```diff
@@ -66,32 +66,39 @@ private DateRange createDateRange(String startDate, String endDate) {
         return dateRange;
     }
 
+    private DateRange createDataRangeWithEqualStartEndDate(String date) {
+        return createDateRange(date, date);
+    }
+
     /**
-     * Creates the Metric object with its required values. Metric is a termin coming from Google Analytics Reporting API.
+     * Creates the Metric objects with its required values. Metric is a termin coming from Google Analytics Reporting API.
+     * Maximum number of metrics for a GA Request is 10
      *
-     * @return the Metric object
+     * @return List of Metric objects
      */
-    private Metric createMetrics() {
-        return new Metric()
-                .setExpression(GA_PAGE_VIEWS.toString())
-                .setExpression(GA_UNIQUE_PAGE_VIEWS.toString())
-                .setExpression(GA_SESSION.toString())
-                .setExpression(GA_BOUNCES.toString())
-                .setExpression(GA_BOUNCE_RATE.toString())
-                .setExpression(GA_AVG_SESSION_DURATION.toString())
-                .setExpression(GA_AVG_TIME_ON_PAGE.toString());
+    private List<Metric> createMetrics() {
```

Comment:
Why is there another implementation in my memory? I thought you have a for-loop where you read the whole enum instead of "import" every single value. I would prefer the loop-way because adding new or removing old metrics means just to edit the enum.